tests/App/DataFixtures/ORM/LoadUserWithServiceData.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Liip/FunctionalTestBundle
+ *
+ * (c) Lukas Kahwe Smith <smith@pooteeweet.org>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Liip\FunctionalTestBundle\Tests\App\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Liip\FunctionalTestBundle\Tests\App\Entity\User;
+use Symfony\Component\Security\Csrf\TokenStorage\TokenStorageInterface;
+
+class LoadUserWithServiceData extends AbstractFixture implements FixtureInterface
+{
+    private $tokenStorage;
+
+    public function __construct(TokenStorageInterface $tokenStorage)
+    {
+        $this->tokenStorage = $tokenStorage;
```

Comment:
Is there a way to use `$this->tokenStorage` in this test? I would like to ensure that this service is callable.

Reply:
I've added a call in the `load` method.